planetarypraxis/smartforests
# feat: restrict contributor tags to one of a set

## Description
Restricts the tags that can be assigned to a contributor to (a) only one and (b) from a set defined in the parent Contributor Index Page.

## Motivation and Context
https://linear.app/commonknowledge/issue/SF3-42/add-sections-on-contributors-page

## How Can It Be Tested?
1. Check the existing contributors that have been tagged are still tagged in the admin system.
2. Change the tags of a contributor and check they are in the right section on the contributors page.

## How Will This Be Deployed?
Normal deployment.
